src/group_theory/perm/sign.lean

```diff
@@ -26,18 +27,19 @@ def of_subtype {p : α → Prop} [decidable_pred p] : perm (subtype p) →* perm
 { to_fun := λ f,
   ⟨λ x, if h : p x then f ⟨x, h⟩ else x, λ x, if h : p x then f⁻¹ ⟨x, h⟩ else x,
   λ x, have h : ∀ h : p x, p (f ⟨x, h⟩), from λ h, (f ⟨x, h⟩).2,
-    by simp; split_ifs at *; simp * at *,
+    by simp only []; split_ifs at *; simp * at *,
```

Comment:
Is the second simp on this line also squeezable?

Reply:
Yes.